front/client/components/ArticleContent.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <div class="article-item">
+    <div class="article-header">
+      <p class="article-hd-textlg">{{ kind }}</p>
+      <a-link openInNewTab :url="`http://hackage.haskell.org/package/${kind}`" class="article-header-link">
+        (Hackage)
+      </a-link>
+      <p class="article-hd-textsm">{{ group }}</p>
+      <div class="article-header-icons">
+        <i class="fas fa-arrow-up"></i>
+        <i class="fas fa-arrow-down"></i>
+        <div class="header-func-icons">
+          <i class="fas fa-cogs"></i>
+          <i class="fas fa-times"></i>
+        </div>
+      </div>
+    </div>
+    <div class="article-content">
+      <p class="article-section-title">Summary</p>
+      <div class="article-description" v-html="itemDescription"></div>
+      <div class="flex-wrapper article-section pros-cons-box">
+        <div class="width-50">
+          <p class="article-section-title">Pros</p>
+          <ul v-if="pros" v-for="(value, index) in pros" :key="index">
+            <li v-html="value.content.html"></li>
+          </ul>
+        </div>
+        <div class="width-50">
+          <p class="article-section-title">Cons</p>
+          <ul v-if="cons" v-for="(value, index) in cons" :key="index">
+            <li v-html="value.content.html"></li>
+          </ul>
+        </div>
+      </div>
+      <div class="article-section">
+        <p class="article-section-title">Ecosystem</p>
+        <div v-html="ecosystem"></div>
+      </div>
+      <div class="article-section notes-box">
+        <p class="article-section-title">Notes</p>
+        <div class="notes-settings">
+          <!-- TODO change a to vue markup element -->
+          <button href="" class="notes-settings-btn" @click="expandNotes()">expand notes</button>
```

Comment:
`@click="expandNotes()` change to `@click="expandNotes"` And so on in other places where you want some function to trigger on event